### DISCUSSION TITLE
Congrats Supabase on your launch! 🥳

### DISCUSSION THREAD

**Discussion by inbytesdevteam** [supabase/supabase#1034] [General]
Saw that you have officially launched Supabase this week and we just want to congratulate the Supabase team!!!! 👏👏👏
We wish Supabase success and more awesome projects built with it! 🙌

🥂, Inbytes.dev Team 😊
![Cheers](https://media.giphy.com/media/tB4OrTLapWnaG73C0f/giphy.gif)

**Comment by kiwicopple** [comment]
thanks @inbytesdevteam ❤️ . 

the team is really happy with the response from the community. We're looking forward to improving it all over time, hopefully making it even easier for developers to use
